api/external/applications/applications.proto

```diff
@@ -211,59 +389,90 @@ message GetDeleteDisconnectedServicesConfigReq{}
+  // Chef Habitat channel that the service is subscribed to.
   string channel = 9;
+  // Update strategy that the service employs.
   string update_strategy = 10;
+  // Site reported by Chef Habitat service, a user defined flag.
   string site = 17;
+  // Intentionally blank.
   HealthStatus previous_health_check = 18;
+  // Time interval of current health status from last status change until now.
   string current_health_since = 19;
+  // Timestamp since health status change.
   google.protobuf.Timestamp health_updated_at = 20;
+  // Service connection information.
+  // based on time since last healthcheck received and disconnected service configuration
   bool disconnected = 21;
+  // Timestamp of last received health check message.
   google.protobuf.Timestamp last_event_occurred_at = 22;
+  // Interval since last event received until now.
   string last_event_since = 23;
   HealthCheckResult health_check_result = 24;
 }
 
+// Request message for GetServiceGroupsHealthCounts
 message ServiceGroupsHealthCountsReq {
+  // Applies search filters, in the format of `fieldname:value`.
   repeated string filter = 1;
 }
+
+// Request message for GetServiceGroups
 message ServiceGroupsReq {
+  // Applies search and status filters, in the format of `fieldname:value` or `status:value`.
   repeated string filter = 1;
+  // Pagination parameters for service groups list.
   common.query.Pagination pagination = 2;
+  // Sorting parameters for service groups list.
   common.query.Sorting sorting = 3;
 }
 
-// A service group message is the representation of one single service group that
-// is internally generated by aggregating all the services
+// A service group message is the representation of an individual service group that
+// is internally generated by aggregating all of its services.
 message ServiceGroup {
+  // Name of service group.
   string name = 1;
   // Combination of the version and release in a single string like:
-  // Example: 0.1.0/8743278934278923
+  // Example: 0.1.0/8743278934278923.
   string release = 2;
+  // Intentionally blank.
   HealthStatus status = 3;
-
-  // The health_percentage can be a number between 0-100
+  // Percentage of services reporting OK status.
+  // The health_percentage can be a number between 0-100.
   int32 health_percentage = 4;
-
+  // Intentionally blank.
   HealthCounts services_health_counts = 5;
+  // Service group ID. This is a value constructed by Chef Automate and is not reported by Chef Habitat.
   string id = 6;
+  // Application name for the service group.
   string application = 7;
+  // Environment name for the service group.
   string environment = 8;
-
   // Combination of the origin and package name in a single string like:
-  // Example: core/redis
+  // Example: core/redis.
   string package = 9;
-
+  // Count of disconnected services within this service group.
   int32 disconnected_count = 10;
 }
 
+// Combined count values from the health status and disconnected status reports.
```

Comment:
The fields of this message do not have descriptions, not sure if that is intentional/important:

![Screen Shot 2019-12-10 at 5 18 36 PM](https://user-images.githubusercontent.com/22962/70582955-1ffab580-1b71-11ea-82f9-55a1c212cc66.png)

Reply:
I'm not sure if the information just isn't bubbling up correctly, or if it isn't defined.